src/log4net.Tests/Util/CyclicBufferTest.cs

```diff
@@ -33,49 +33,47 @@ namespace log4net.Tests.Util;
 public class CyclicBufferTest
 {
   [Test]
-  public void TestConstructorSize0()
-  {
-    Assert.Throws<ArgumentOutOfRangeException>(() => new CyclicBuffer(0));
-  }
+  public void TestConstructorSize0() 
+    => Assert.Throws<ArgumentOutOfRangeException>(() => new CyclicBuffer(0));
```

Comment:
I know that a lot of the Assert.* assertions were removed in favor of using `Assert.That` - there's an `Assert.That` for verifying that things do or don't throw as well, eg:

```csharp
            Assert.That(
                () => throw new InvalidOperationException("123"),
                Throws.Exception.InstanceOf<InvalidOperationException>()
                    .With.Message.EqualTo("123")
            );
```

it seems like `Assert.Throws` is still valid, but I wonder for how long?

Reply:
Thanks for the hint. I've adjusted the tests.